src/App.js

```diff
@@ -9,6 +9,18 @@ export class App extends Component {
     movies: data,
   };
 
+  addMovie = (newMovie) => {
+    const { movies } = this.state;
+    const isMovieInData = movies
+      .some(movie => movie.imdbId === newMovie.imdbId);
+
+    if (!isMovieInData && newMovie) {
```

Comment:
```suggestion
    const isMovieUnique = movies
      .every(movie => movie.imdbId !== newMovie.imdbId);

    if (isMovieUnique && newMovie) {
```
It's easier to read positive condition than a negative one